package.json

```diff
@@ -9,10 +9,17 @@
     "docs": "yarn workspace @thinkmill/monorepo-docs",
     "website:build": "yarn website build",
     "website:start": "yarn docs:format && yarn website start",
-    "docs:format": "yarn docs format"
+    "docs:format": "yarn docs format",
+    "lint": "yarn eslint . --ext .ts,.tsx,.js"
```

Comment:
try removing the `.` before the extensions, `"lint": "yarn eslint . --ext ts,tsx,js"`